reports/performance/_outliers.qmd

```diff
@@ -368,3 +368,133 @@ qnt_df <- assessment_pin %>%
 qnt_df %>%
   datatable(rownames = FALSE)
 ```
+
+
+## Increase in Housing Value
+
+This section looks at the year over year change in assessed housing values. It separates the data into individual observations, identifying 10 PINs in each neighborhood which grew at the fastest rate as well as the median rate of change for each neighborhood. 
+
+::: panel-tabset
+
+## Map of Largest YOY Changes
+
+This map caps the legend at a 300% to increase legibility. It then plots the 10 largest year over year increases in assessed values by neighborhood. 
+
+```{r _outliers_map_of_largest_change}
```

Comment:
A lot of the ultra large % changes here are going to be things going from like $1,000 to $100K. Some of these are errors, but some are new development on vacant land. It might be good to dig on this a little bit and find some heuristics to try to identify _suspicious_ AV changes, rather than just large ones.